pallets/liquid-staking/src/lib.rs

```diff
@@ -464,15 +447,15 @@ pub mod pallet {
 
         /// Update default xcm fees
         /// it reflects xcm fees consumed on relaychain
-        #[pallet::weight(<T as Config>::WeightInfo::update_xcm_fees_compensation())]
+        #[pallet::weight(<T as Config>::WeightInfo::update_xcm_fees())]
         #[transactional]
-        pub fn update_xcm_fees_compensation(
+        pub fn update_xcm_fees(
```

Comment:
update_xcm_fees should be moved totally to pallet-xcm-helper

Reply:
This is similar to crowd loan pallet https://github.com/parallel-finance/parallel/blob/master/pallets/crowdloans/src/lib.rs#L544

Are you suggesting to move this dispatchable function completely in xcm-helper from crowd loan and liquid-staking?

Reply:
yes because otherwise we will have two pallets which basically mutate the same storage

Reply:
sure we will do it as a part of #1053